lib/cc/analyzer/issue_validations/remediation_points_validation.rb

```diff
@@ -0,0 +1,25 @@
+module CC
+  module Analyzer
+    module IssueValidations
+      class RemediationPointsValidation < Validation
+        def valid?
+          remediation_points.nil? || positive_integer?(remediation_points)
+        end
+
+        def message
+          "Remediation points must be a positive integer"
```

Comment:
should we use the terminology `non-negative` here rather than `positive` since it can be zero?

Reply:
Done.